reflex/utils/format.py

```diff
@@ -300,9 +301,21 @@ def format_prop(
 
         # Handle event props.
         elif isinstance(prop, EventChain):
+            if prop.args_spec is None:
```

Comment:
Maybe we can move this into a function `format_event_chain` as it's quite complex? (Can be done in a follow up)

Reply:
This is the code for the backward compatibility of previous EVENT_ARG, we can simplify it later when we actually remove the feature.